caf-logging-serilog/README.md

```diff
@@ -0,0 +1,67 @@
+# CAF Logging Serilog
+This project configures Serilog to meet the CAF Logging Standard.
+
+The log level is controllable via value that can be provided on Logger initialization. The default used is `INFO`.
+
+All logs are being redirected to `stderr`.
+A consistent log message format is used.
+
+The header information is in square brackets.
+Fixed size columns are used so that messages align vertically.
+Missing or inapplicable header fields are replaced with a dash.
+Log Injection attacks are prevented.
+
+Any unsafe characters found in the Tenant Id, Correlation Id, or Logger are removed.
+The log message is JSON-encoded if it contains any control characters (including newlines) or any non-ASCII characters.
+Exception stack traces are JSON-encoded so that they do not span multiple lines.
+
+## Dependency
+
+Add the nuget package `MicroFocus.CafApi.CafLoggingSerilog`
+
+## Pattern
+
+`[{@t:yyyy-MM-dd HH:mm:ss.fffZ} {Tid(ProcessId,ThreadId)} {Log(@l):5} {Sanitize(tenantId, 12, 12)} {Sanitize(correlationId, 4, 4)}] {Sanitize(logger, 30, 30)}: {MaybeJsonMsgAndEx(@m,@x)}\n`
+
+## Examples
+
+`[2022-05-10 13:29:45.444Z #217.001 WARN  john-tenant  Cidi] c.m.d.f.f.f.FieldFullTextFixer: "Greek test: κόσμε"`
```

Comment:
Why does the strings being logged have quotes around them?  And also why are there extra quotes in the strings in the `message` field?

I.e. I tried this: `Log.Error(ex, "Caught an exception");`

I expected: `{"message":"Caught an exception","exception":"..."}`
But I got: `{"message":"\"Caught an exception\"","exception":"..."}`

Reply:
Fixed